catalog/policies/git_push.default.rego

```diff
@@ -60,8 +61,8 @@ project_root := input.stack.project_root
 # https://www.openpolicyagent.org/docs/latest/policy-reference/#iteration
 project_affected {
     some i, j
-    startswith(affected_files[i], project_root)
-    endswith(affected_files[i], tracked_extensions[j])
+    strings.any_prefix_match(affected_files[i], project_root)
+    strings.any_suffix_match(affected_files[i], tracked_extensions[j])
```

Comment:
The functions accept lists, so this can just be
```
project_affected {
  strings.any_prefix_match(affected_files, project_root)
  strings.any_suffix_match(affected_files, tracked_extensions)
}
```
and this is the approach that gets the huge speedup.

Reply:
This is correct (don't have access to resolve the comment).